### ISSUE TITLE
[jsdoc/require-description-complete-sentence] Error on @link type

### ISSUE THREAD

**Issue by pyoor** [gajus/eslint-plugin-jsdoc#705]
## Expected behavior
The plugin `jsdoc/require-description-complete-sentence` would not raise when including a `@link` types.

## Actual behavior
An error is raised.

## ESLint Config
```js
module.exports = {
  env: {
    es2020: true,
    node: true,
  },
  root: true,
  parser: '@typescript-eslint/parser',
  plugins: ['@typescript-eslint', 'jsdoc'],
  rules: {
    'jsdoc/require-description-complete-sentence': ['warn'],
  },
}
```

## ESLint sample
```js
/**
 * Say Hello.
 *
 * {@link http://www.github.com} // ESLint: Sentence must end with a period.(jsdoc/require-description-complete-sentence)
 */
function hello(): void {
    console.log("hello world")
}
```

## Environment

- Node version: v14.16.0
- ESLint version v7.22.0
- `eslint-plugin-jsdoc` version: v32.3.0

**Comment by brettz9** [comment]
One problem here is that `@link` is inline, so a person may actually want a period after it.

**Comment by brettz9** [comment]
Closing as I think insisting on a sentence would call for a period after the inline link too.
